ClientLib/src/main/java/com/o3dr/services/android/lib/drone/mission/item/command/Takeoff.kt

```diff
@@ -0,0 +1,86 @@
+package com.o3dr.services.android.lib.drone.mission.item.command
+
+import com.o3dr.services.android.lib.drone.mission.item.MissionItem
+import android.os.Parcelable
+import com.o3dr.services.android.lib.drone.mission.item.command.Takeoff
+import android.os.Parcel
+import com.o3dr.services.android.lib.drone.mission.MissionItemType
+
+/**
+ * The vehicle will climb straight up from it’s current location to the altitude specified (in meters).
+ * This should be the first command of nearly all missions.
+ * If the mission is begun while the copter is already flying, the vehicle will climb straight up to the specified altitude.
+ * If the vehicle is already above the specified altitude the takeoff command will be ignored and the mission will move onto the next command immediately.
+ *
+ * Created by fhuya on 11/6/14.
+ */
+class Takeoff : MissionItem, MissionItem.Command, Parcelable {
+    /**
+     * @return take off altitude in meters
+     */
+    /**
+     * Sets the take off altitude
```

Comment:
More auto formatted documentation

Reply:
P.S. I know these are just from the Java->Kotlin auto converter and it doesn't hurt anything. I'm just thinking we might as well fix since I'm going through all the code